src/components/RightSideBar/CurrencyConverterComponent/CurrencyInput.jsx

```diff
@@ -0,0 +1,32 @@
+import PropTypes from "prop-types";
+import "./currencyInput.css";
+
+function CurrencyInput(props) {
+  return (
+    <div className="group">
+      <input
+        type="text"
+        value={props.amount}
+        onChange={(ev) => props.onAmountChange(ev.target.value)}
```

Comment:
Create separate functions and call them here

Reply:
Sir I tried here but not done